improver/precipitation_type/.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
There are a few IDE-generated files in this .idea directory. Suggest removing these as IDE configuration files generally shouldn't be in version control.